core-bundle/src/EventListener/DataContainer/DisableAppConfiguredSettingsListener.php

```diff
@@ -27,21 +28,26 @@ public function __construct(
     ) {
     }
 
-    public function onLoadCallback(): void
+    public function onLoadCallback(DataContainer|null $dc = null): void
     {
+        // return if there is no table
+        if (null === $table = $dc?->table) {
```

Comment:
Nitpick: could be shortened to

```suggestion
        if (!$table = $dc?->table) {
```